setup.py

```diff
@@ -492,6 +493,8 @@ def pxd(name):
                      'sources': ['pandas/_libs/src/datetime/np_datetime.c',
                                  'pandas/_libs/src/datetime/np_datetime_strings.c',
                                  'pandas/_libs/src/period_helper.c']},
+    '_libs.tslibs.frequencies': {'pyxfile': '_libs/tslibs/frequencies',
```

Comment:
this *also* needs to be listed in the ``setup`` function, IOW all sub-dirs have to be listed, put it after ``pandas._libs``